internal/os/volume/api.go

```diff
@@ -109,6 +110,27 @@ func (VolAPIImplementor) ResizeVolume(volumeID string, size int64) error {
 		finalSize = size
 	}
 
+	cmd = fmt.Sprintf("Get-Volume -UniqueId \"%s\" | Get-partition | Select Size", volumeID)
+	out, err = runExec(cmd)
+
+	if err != nil || len(out) == 0 {
+		return fmt.Errorf("error getting size of the partition from moun. cmd %s, output: %s, error: %v", cmd, string(out), err)
+	}
+
+	var getPartitionSizing map[string]int64
+	outString = string(out)
+	err = json.Unmarshal([]byte(outString), &getPartitionSizing)
```

Comment:
instead of using Unmarshal, could we use e.g., 
```
 (Get-Volume -UniqueId \"%s\"  | Get-Partition).Size
strings.TrimSpace(string(out))
```

Reply:
done